backend/src/main/java/gov/cdc/usds/simplereport/config/DataLoaders.java

```diff
@@ -0,0 +1,64 @@
+package gov.cdc.usds.simplereport.config;
+
+import static java.util.concurrent.CompletableFuture.supplyAsync;
+
+import gov.cdc.usds.simplereport.db.model.PatientPreferences;
+import gov.cdc.usds.simplereport.db.model.TestEvent;
+import gov.cdc.usds.simplereport.db.repository.PatientPreferencesRepository;
+import gov.cdc.usds.simplereport.db.repository.TestEventRepository;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import org.dataloader.DataLoader;
+import org.dataloader.DataLoaderRegistry;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DataLoaders {
+  private final PatientPreferencesRepository patientPreferencesRepository;
+  private final TestEventRepository testEventRepository;
+  public static final String PATIENT_LAST_TEST = "patientLastTestLoader";
+  public static final String PATIENT_PREFERENCES = "patientPreferencesLoader";
```

Comment:
Not super important, but should these names reflect the query that they support? E.g., `patients[*].lastTest` and `patients[*].preferences` instead of `patientLastTestLoader` or `patientPreferencesLoader`, respectively.

Reply:
They're just Strings that key the map, so I don't see a downside here. Done in 0159df39